calendar-aggregator-test/src/test/java/io/github/dannyflowerz/calendaraggregator/ft/stepdefinitions/GetAppointmentsStepDefs.java

```diff
@@ -21,42 +9,62 @@
 import io.github.dannyflowerz.calendaraggregator.model.OutlookAppointment;
 import io.restassured.RestAssured;
 import io.restassured.response.Response;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.junit.Assert.assertEquals;
 
 public class GetAppointmentsStepDefs implements En {
 
     private static Gson GSON = new GsonBuilder().setDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'").create();
     private String startDate;
     private String endDate;
     private Response response;
+    private WireMockServer wireMockServer = new WireMockServer();
+    private CloseableHttpClient httpClient = HttpClients.createDefault();
 
-	public GetAppointmentsStepDefs() {
-		Before(() -> {
-            configureFor(9081);
+    public GetAppointmentsStepDefs() {
+        Before(() -> {
+            wireMockServer.start();
```

Comment:
You can assume that the application is started with the "stubbed" spring profile, so you can just connect to the running wiremock server as in the original version of the code